MFVNeutralino/test/TrackMover/efficiencies.py

```diff
@@ -27,8 +27,9 @@ def __init__(self, **kwargs):
                                   cutset = ['nocuts', 'ntracks', 'all'],
                                   nsigmadxy = [4.0],
                                   tau = [100,300,1000,10000,30000],
-                                  njets = [2,3],
-                                  nbjets = [0,1,2],
+                                  njets = [2],
+                                  nbjets = [0],
+                                  year = [2017], #FIXME 
```

Comment:
Out of curiosity, is this the only place that a change is needed to switch from one year to another for this study? I'm sort of surprised it's here at all!